errors.go

```diff
@@ -66,4 +66,14 @@ var (
 		Code:    110,
 		Message: "expired",
 	}
+	// ErrTokenInvalid ...
```

Comment:
Not what I suggested actually - sorry that I explained bad:) This is a list of errors that can be returned to client in client-server protocol. If you can see they all start with `Error...`, not from `Err...`. The idea was to introduce new internal errors in `token_verifier.go` file:
```
ErrTokenInvalid = errors.New("...")
ErrTokenInvalidInfo = errors.New("...")
ErrTokenExpired = errors.New("...")
``` 

And use them.

BTW yesterday I realized that as you personally don't need this feature yourself then we can simply do refactoring but not export this API publicly. This will give us more freedom to change things in future without breaking users code. I suppose we already solved an original case why this JWT issue was raised (support RSA tokens) so we can safely wait for other request from library user that will need this as part of public API.  

So what I suggest is make all these changes private for now. This is a wonderful refactoring that I really like but I think if we keep a possibility to not extend public API at all then we must use this chance until there will be request from user. This means that we can keep these new errors private for now:

```
errTokenInvalid = errors.New("...")
errTokenInvalidInfo = errors.New("...")
errTokenExpired = errors.New("...")
```

The same for all other entities in pull request:

```
TokenVerifier
ConnectToken
SubscribeToken
```

Then it will be very simple to make this API public.